salt/states/service.py

```diff
@@ -549,7 +554,12 @@ def dead(name,
     # command, so it is just an indicator but can not be fully trusted
     before_toggle_status = __salt__['service.status'](name, sig)
     if 'service.enabled' in __salt__:
-        before_toggle_enable_status = __salt__['service.enabled'](name)
+        if salt.utils.is_windows():
+            # service.enabled in Windows returns True for services that are set
+            # to Auto start, but services set to Manual can also be disabled
+            before_toggle_enable_status = __salt__['service.info'](name)['StartType'] in ['Auto', 'Manual']
```

Comment:
Can we create a function called `_salt__['service.enabled'](name)` rather than call `__salt__['service.info'](name)['StartType']  in 'Auto', Manual'`.  Keep in interface consistent across platforms.

Reply:
Same issue as noted above.